sys-apps/ignition/files/README.md

```diff
@@ -0,0 +1,41 @@
+# Ignition patches
+
+These patches bring Flatcar custom features (from a runtime execution point-of-view) to CoreOS's Ignition.
+
+## Sync Ignition with the upstream
+
+When a new upstream Ignition release is out, we need to backport the Flatcar patches on this new version.
+
+For this, we can just apply the current patches and fix any broken stuffs:
+
+```bash
+git clone https://github.com/coreos/ignition
+cd ignition
+git checkout "${RELEASE}"
+git am /path/to/coreos-overlay/sys-apps/ignition/files/00*
+```
+
+Once done, we can generate the new set of patches:
+```
+git format-patch "${RELEASE}"
+```
+
+Copy the new patches to `::coreos-overlay`
+```
+cp 00* /path/to/coreos-overlay/sys-apps/ignition/files/
+```
+
+:warning: We might need to update the applied patches name in the Ignition's ebuild.
+
+## Ignition converter (ign-converter)
+
+This converter is the central piece of auto-translation from Ignition 2 to Ignition 3. While we still support both version, we need to maintain it. It was initially on `coreos/ign-converter` then it has moved to `flatcar/ign-converter` to apply some logic (that could be upstreamed).
```

Comment:
```suggestion
This converter is the central piece of auto-translation from Ignition config version 2 to Ignition config version 3. While we still support both version, we need to maintain it. It was initially on `coreos/ign-converter` then it has moved to `flatcar/ign-converter` to apply some logic (that could be upstreamed).
```

Reply:
Sidenote: The patch probably also only works because we lift to 3.1 and not 3.4, otherwise we would miss the `if version == types.MaxVersion {` check.